cloudflare/resource_cloudflare_worker_script_test.go

```diff
@@ -74,24 +74,29 @@ resource "cloudflare_worker_script" "%[1]s" {
 }`, rnd, scriptContent2)
 }
 
-func testAccCheckCloudflareWorkerScriptConfigMultiScriptUpdateKvNamespaceBinding(rnd string) string {
+func testAccCheckCloudflareWorkerScriptConfigMultiScriptUpdateBinding(rnd string) string {
 	return fmt.Sprintf(`
 resource "cloudflare_workers_kv_namespace" "%[1]s" {
- title = "%[1]s"
+  title = "%[1]s"
 }
 
 resource "cloudflare_worker_script" "%[1]s" {
-  name = "%[1]s"
+  name    = "%[1]s"
   content = "%[2]s"
 
   kv_namespace_binding {
-	name = "%[1]s"
-	namespace_id = cloudflare_workers_kv_namespace.%[1]s.id
+    name         = "KV_NAMESPACE"
+    namespace_id = cloudflare_workers_kv_namespace.%[1]s.id
   }
 
-  kv_namespace_binding {
-	name = "%[1]s-copy"
-	namespace_id = cloudflare_workers_kv_namespace.%[1]s.id
+  plain_text_binding {
+    name = "PLAIN_TEXT"
```

Comment:
Just to confirm here,  PLAIN_TEXT is just the arbitrary name of the key, right? If that’s the case, can we change the example to something more example like? That way we know it’s definitely an arbitrary key.

Reply:
Is this comment about documentation or the test case? I updated the documentation, I can also change the test cases if it helps.

Reply:
Both. Ideally I’d like to remove any question whether this is a special value (such as a constant) or whether the field can be something arbitrary

Reply:
Do you want to specify the behavior? I don't want to make it any more confusing, it seemed clearest to me because these are the keys checked in the test suite so if one is missing you know which it is easily. If this is about the casing, I can change it, but that's the standard style I've seen and used in workers because it binds to a constant in JavaScript.

Reply:
Can you comment on whether the documentation change is clear enough?

Reply:
Tests I don't mind but I've just proposed (hopefully) very obvious example values to not confuse the documentation readers.

Reply:
Ok, I've updated the documentation with your suggestions minus the `_KEY` suffix. I removed it because it's not really a key to something, but a direct binding in the script as a variable. I can add it back if you think it's still helpful though.